### PR TITLE
Upgrade to storybook 5.2.0

### PR DESCRIPTION
**Pull request by jorenbroekema**
Sadly I couldn't yet include the awesome documentation feature that addon-docgen brings, because it is not compatible yet with @storybook/polymer :(. 

Once it is, I am happy to add some stuff for it to @open-wc docs, how to use, include it in the demo and the storybook scaffold, etc.

For now, upgrading seems like a nice thing already ^^

### CODE REVIEW

File: packages/demoing-storybook/default-storybook-webpack-config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = ({ config, transpilePackages = ['lit-html', 'lit-element', '@open-wc'] }) => {
   config.module.rules.push({
     test: [/\.stories\.js$/, /index\.js$/],
-    loaders: [require.resolve('@storybook/addon-storysource/loader')],
+    loaders: [require.resolve('@storybook/source-loader')],
```
Comment (jorenbroekema) [inline packages/demoing-storybook/default-storybook-webpack-config.js]: This is the new way, old one is deprecated (gives you a warning in CLI)

File: packages/demoing-storybook/demo/.storybook/.babelrc [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-    "@babel/plugin-proposal-object-rest-spread"
-  ],
+  "plugins": ["@babel/plugin-syntax-dynamic-import", "@babel/plugin-proposal-object-rest-spread"],
   "presets": [
     [
       "@babel/preset-env",
       {
-        "useBuiltIns": "entry"
+        "useBuiltIns": "entry",
+        "corejs": "2"
```
Comment (jorenbroekema) [inline packages/demoing-storybook/demo/.storybook/.babelrc]: Added this to get rid of warning message in CLI

**Comment by jorenbroekema** [comment]
I generated a new lock file btw, not sure if this is what you guys want :P

**Comment by jorenbroekema** [comment]
Seems like headers are failing due to some core-js issue, maybe it has to do with the codebase mixing v2 and v3 and folder names are different in v3?

As for tests, same ones are failing on master sadly.

**Comment by bennypowers** [comment]
Take a look at https://github.com/storybookjs/storybook/issues/7084#issuecomment-522883376, might help you to get the build going

**Comment by jorenbroekema** [comment]
Aah I see. Do you know which part of your config was the crux in having it fail / having it succeed?

**Comment by bennypowers** [comment]
I had two major issues

1) excluding babel from transpilation
```js
const BABEL_LOADER_RULE = {
  test: /\.js$/,
  loader: 'babel-loader',
  exclude: [/@babel\/runtime/, /core-js/, /axe-core/],
  options
}
```
2) issues with polymer-webpack-loader (see https://github.com/storybookjs/storybook/issues/7986)

**Comment by daKmoR** [comment]
thx for looking at it 🤗 

going to close this as we now moved to storybook 5.3 of`@storybook/web-components` which brings multiple other benefits as well 👍

**Pull request closed without merging**